static/js/ducks/followup_requests.js

```diff
@@ -25,9 +27,18 @@ export const prioritizeFollowupRequests = (params = {}) =>
     params
   );
 
-messageHandler.add((actionType, payload, dispatch) => {
+messageHandler.add((actionType, payload, dispatch, getState) => {
+  const { source } = getState();
+
   if (actionType === REFRESH_FOLLOWUP_REQUESTS) {
-    dispatch(fetchFollowupRequests());
+    const loaded_obj_key = source?.internal_key;
+    if ("obj_key" in payload) {
+      if (loaded_obj_key === payload.obj_key) {
+        dispatch(sourceAction.fetchSource(source.id));
+      }
+    } else {
+      dispatch(fetchFollowupRequests());
```

Comment:
This doesn't seem to fit with the rest of the pattern here.

Reply:
Yeah reverting to the old way.